src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -153,6 +161,24 @@ namespace AppInstaller::Manifest
                 resultErrors.emplace_back(ManifestError::ExeInstallerMissingSilentSwitches, ValidationError::Level::Warning);
             }
 
+            if (installer.InstallerType == InstallerTypeEnum::Portable)
+            {
+                if ((manifest.DefaultInstallerInfo.AppsAndFeaturesEntries.size() > 1 && installer.AppsAndFeaturesEntries.size() == 0) ||

```

Comment:
>manifest.DefaultInstallerInfo.AppsAndFeaturesEntries.size() > 1 && installer.AppsAndFeaturesEntries.size() == 0) [](http://example.com/codeflow?start=21&length=112)

This condition does not seem possible, installer level fields directly inherit from DefaultInstallerInfo. If you really need to verify this, just verify the installer level directly. This means we are putting a burden on manifest author to always override portable installer with empty ones.

Same for below 2

Reply:
Removed additional condition and only check value at the installer level directly.